.github/workflows/test-suite.yml

```diff
@@ -52,10 +52,10 @@ jobs:
       provider_version: ${{ matrix.provider_version }}
 
   acc-tests:
-    needs: [mig-tests, versions]
+    needs: [clean-before, versions]
```

Comment:
why the change from `mig-tests` to `clean-before`? I see that you added `mig-tests` in `clean-after` but not in the `clean-before`

Reply:
so mig and acc run in parallel

Reply:
mig_test already had:

 mig-tests:
    needs: [clean-before, versions]